### DISCUSSION TITLE
HDR support in MAME?

### DISCUSSION THREAD

**Discussion by john-iv** [mamedev/discussions#57] [Ideas]
Probably for @cuavas @mooglyguy, but was curious if HDR could be supported in MAME as a way to, forgive my imprecise language, boost/juice the brightness of the raster/vector image up a bunch of nits.

One thing we get with the shaders is a somewhat darkening of the screen, could this be overcome with an application of HDR to it? The original CRTs are quite bright in comparison to our modern LCDs.

I recall this was mentioned 5+ years ago when HDR monitors were not as common as today; but was curious to see if the tech. has matured since or if there are philosophical/technical reasons for disallowing it.  Thanks -

**Comment by rb6502** [comment]
We're not opposed and I think it will be necessary for good vector rendering in the same way that VRR is accepted as the real fix for tearing, but I don't think it's something that can just be bolted on to the existing 2006-era rendering architecture.  @MooglyGuy would probably know more though.
